### ISSUE TITLE
When I try show slash menu on production, always shows e.fn is not a function error

### ISSUE THREAD

**Issue by nabi-chan** [TypeCellOS/BlockNote#276]
**Describe the bug**

When I try showing any menu (slash, Toolbar, SideMenu) on production, this error always shown,
It seems tippyjs issue... but I didn't catch any clue, so I wrote this issue on this repo

<img width="583" alt="스크린샷 2023-07-19 13 09 31" src="https://github.com/TypeCellOS/BlockNote/assets/42037851/24bbb5d0-fd35-449c-a339-b3f3a45a6ebf">

(I added some source map for more clue)

**To Reproduce**
1. [clone this GitHub code](https://github.com/nabi-chan/thermal-printer)
2. run `pnpm build && pnpm start`
3. try slash menu or popup toolbar on browser

**Misc**
- Node version: v18.16.1
- Package manager: pnpm@8.6.8
- Browser: chrome chrome@114.0.5735.198

**Comment by YousefED** [comment]
Thanks @nabi-chan for this, and thanks for your sponsorship! Very much appreciated.

We're currently refactoring how the menus / tippy instances get rendered. See #270 . We hope to get this released next week! cc @matthewlipski

**Comment by joshua-lawrence** [comment]
Just wanted to add that we are also encountering this issue in our production build!

**Comment by dlwocks31** [comment]
We are also facing this same issue. Looking forward to the rearchitecture!

**Comment by joshua-lawrence** [comment]
Running on Next.js 13 here, its appears the issue is still present for me:
![image](https://github.com/TypeCellOS/BlockNote/assets/25112774/61518546-799b-4612-99af-5cc986f7b78b)

Behavior is slightly different now though. The page doesn't compile anymore, I'm getting my error page. I'll investigate and see if I can help find a root cause.

**Comment by joshua-lawrence** [comment]
There is an empty object in the Tippy plugins array:

![Screenshot 2023-08-02 at 11 25 10 AM](https://github.com/TypeCellOS/BlockNote/assets/25112774/83158fd6-6a2f-4d11-8657-8c8be40bd55a)

[The particular function above that's throwing the error in @tippyjs/react](https://github.com/atomiks/tippyjs/blob/ad85f6feb79cf6c5853c43bf1b2a50c4fa98e7a1/src/createTippy.ts#L136)

Still digging around..

**Comment by YousefED** [comment]
It seems a very weird bug! My guess at this point is the webpack minifier strips away some essential code. Try disabling it like this:

next.config.js:
```
/** @type {import('next').NextConfig} */
const nextConfig = {
  reactStrictMode: true,
  webpack(webpackConfig) {
    return {
      ...webpackConfig,
      optimization: {
        minimize: false,
      },
    };
  },
};

module.exports = nextConfig;
```

does that remove the error?

**Comment by joshua-lawrence** [comment]
@YousefED that didn't seem to work for me. Appears this is a known issue for tippy:
https://github.com/atomiks/tippyjs-react/issues/379#issuecomment-1649100565
https://github.com/vercel/next.js/issues/53151#issuecomment-1655734703

**Comment by joshua-lawrence** [comment]
Oops, realized I set the config incorrectly on webpack. @YousefED
With webpack minification disabled, the JavaScript heap runs out of memory. Given this isn't a BlockNote issue, I'll look elsewhere for solutions and update if I find a workaround that's working for me. Thanks.

**Comment by nabi-chan** [comment]
@joshua-lawrence It fixed on next@13.4.19! Also, I think it was fixed. thanks all for support :)

**Comment by YousefED** [comment]
Great! Thanks for looping back @nabi-chan
